tests/unit/models/generation/test_vllm_generation.py

```
@@ -1039,6 +1046,223 @@ def test_vllm_generate_text(cluster, tokenizer):
+        top_p=generation_config["top_p"],
+        # We want to test the actual train flow and how this is used. So we need to get logprobs here.
+        logprobs=True,
+        return_tokens_as_token_ids=True,
+        max_tokens=1,
+    )
+
+    # Take a short nap for the server to spinup. Maybe there is a better way to do this?
+    sleep(3)
+
+    # Generate and check result
+    response = requests.post(url=f"{base_urls[0]}/chat/completions", json=body)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)